build.py

```diff
@@ -1588,8 +1629,10 @@ def __check_tools(self, opts):
         print_debug("wget: %s" % (self.wget,))
 
         self.nuget = opts.nuget_path
-        if not os.path.exists(self.nuget):
-            print_log("Could not find nuget: %s" % (self.nuget,))
+        if opts.nuget_path:

```

Comment:
So I simplify also the Project_nuget() 'cause we assume we always download/install the file in tools\nuget ? 
I'll also drop all the preprocess() stuff on the singles project because now is useless or I kkep it for the future ?